CMakeLists.txt

```diff
@@ -31,10 +33,8 @@ if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang") AND NOT (CMAKE_CXX_COMPILER_VER
 endif ()
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
-aux_source_directory(./src SRC_LIST)
 file(GLOB_RECURSE SRC_LIST "./plugins/*.cpp" "./src/*.cpp")
 file(GLOB_RECURSE HEADERS "./plugins/*.h" "./src/*.h")
```

Comment:
> `./` is not needed.

one of the cmake versions (don't remember which one now) has problems if you do not put `./` in path.

> Leter the is another `file(GLOB HEADERS ...)`.

fixed in 5b2e6435825c44b6e94640eda065e1bcab0c52a5